<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Account::Internal::MedicalProcedureWidget</class>
 <widget class="QWidget" name="Account::Internal::MedicalProcedureWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>476</width>
    <height>441</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0" colspan="4">
    <widget class="QLabel" name="label">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Available personnal medical procedures</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="6" column="0" colspan="4">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Account details</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="3" column="0" colspan="2">
       <widget class="QLabel" name="nameLabel">
        <property name="text">
         <string>Name</string>
        </property>
        <property name="buddy">
         <cstring>name</cstring>
        </property>
       </widget>
      </item>
      <item row="3" column="2" colspan="2">
       <widget class="QLineEdit" name="name"/>
      </item>
      <item row="4" column="0" colspan="2">
       <widget class="QLabel" name="abstractLabel">
        <property name="text">
         <string>Abstract</string>
        </property>
        <property name="buddy">
         <cstring>abstractEdit</cstring>
        </property>
       </widget>
      </item>
      <item row="4" column="2" colspan="2">
       <widget class="QLineEdit" name="abstractEdit"/>
      </item>
      <item row="1" column="0" colspan="2">
       <widget class="QLabel" name="ownerLabel">
        <property name="text">
         <string>Owner</string>
        </property>
        <property name="buddy">
         <cstring>ownersComboBox</cstring>
        </property>
       </widget>
      </item>
      <item row="5" column="0" colspan="2">
       <widget class="QLabel" name="typeLabel">
        <property name="text">
         <string>Type</string>
        </property>
        <property name="buddy">
         <cstring>type</cstring>
        </property>
       </widget>
      </item>
      <item row="5" column="2" colspan="2">
       <widget class="QLineEdit" name="type"/>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="amountLabel">
        <property name="text">
         <string>Amount</string>
        </property>
        <property name="buddy">
         <cstring>amountSpin</cstring>
        </property>
       </widget>
      </item>
      <item row="6" column="2">
       <widget class="QDoubleSpinBox" name="amountSpin"/>
      </item>
      <item row="1" column="2" colspan="2">
       <widget class="QComboBox" name="ownersComboBox"/>
      </item>
      <item row="2" column="2" colspan="2">
       <widget class="QLabel" name="userUidLabel">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>User UUID</string>
        </property>
       </widget>
      </item>
      <item row="7" column="0">
       <widget class="QLabel" name="reinmbLabel">
        <property name="text">
         <string>Reinbursment rate</string>
        </property>
        <property name="buddy">
         <cstring>rateSpin</cstring>
        </property>
       </widget>
      </item>
      <item row="7" column="2">
       <widget class="QDoubleSpinBox" name="rateSpin"/>
      </item>
      <item row="9" column="0">
       <widget class="QLabel" name="dateLabel">
        <property name="text">
         <string>Date</string>
        </property>
        <property name="buddy">
         <cstring>dateEdit</cstring>
        </property>
       </widget>
      </item>
      <item row="9" column="2">
       <widget class="QDateEdit" name="dateEdit"/>
      </item>
      <item row="11" column="0" colspan="4">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QPushButton" name="save">
          <property name="maximumSize">
           <size>
            <width>100</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>Save</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0" colspan="4">
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="2" column="3">
    <widget class="QPushButton" name="addButton">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="2" column="1" colspan="2">
    <widget class="QComboBox" name="mpComboBox"/>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="MPLabel">
     <property name="text">
      <string>Medical 
Procedure</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
     <property name="buddy">
      <cstring>mpComboBox</cstring>
     </property>
    </widget>
   </item>
   <item row="3" column="3">
    <widget class="QPushButton" name="removeButton">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>name</tabstop>
  <tabstop>abstractEdit</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
